server/requirements.txt

```diff
@@ -1,2 +1,4 @@
 fastapi
 uvicorn
+uvicorn[standard]
```

Comment:
Just to confirm, do we need both of these?

Reply:
`uvicorn[standard]` is a superset of `uvicorn` that includes WebSocket support, so `uvicorn` can be removed from line 2.

Reply:
👍 thanks! Done